src/traces/scatter/defaults.js

```diff
@@ -38,6 +38,7 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
         'lines+markers' : 'lines';
     coerce('text');
     coerce('hovertext');
+    coerce('hovertemplate');
```

Comment:
We shouldn't `coerce` this when `hoveron: 'fills'`

Reply:
Done in 2015c57